.github/workflows/archive-full-index.yaml

```diff
@@ -0,0 +1,146 @@
+name: Generate the full index archive for Bitnami Charts
+on:
+  schedule:
+    - cron: '0 0 * * *'
+env:
+  LAST_INDEX_URL: https://charts.bitnami.com/bitnami/index.yaml
+  INDEXES_DIR: indexes
+jobs:
+  get:
+    runs-on: ubuntu-latest
+    name: Get
+    steps:
+      - id: checkout-repo
+        name: Checkout repo
+        uses: actions/checkout@v3
+      - id: get-last-indexes
+        name: Get indexes
+        run: |
+          mkdir $INDEXES_DIR
+          curl $LAST_INDEX_URL -o ./$INDEXES_DIR/last_index.yaml
+          cp bitnami/index.yaml ./$INDEXES_DIR/old_index.yaml
+      - id: indexes-lint
+        name: Lint indexes
+        run: |
+          cat << EOF > config
+          extends: relaxed
+
+          rules:
+            indentation:
+              level: error
+            line-length: disable
+          EOF
+          yamllint -c config ./$INDEXES_DIR/*index.yaml
+      - id: upload-artifact
+        name: Upload artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: indexes
+          path: ./${{ env.INDEXES_DIR }}
+          retention-days: 2
+          if-no-files-found: error
+  merge:
+    runs-on: ubuntu-latest
+    needs: get
+    name: Merge
+    steps:
+      - id: download-artifact
+        name: Download artifacts
+        uses: actions/download-artifact@v3
+        with:
+          name: indexes
+      - id: merge
+        name: Merge
+        run: yq eval-all '. as $item ireduce ({}; . *+ $item )' old_index.yaml last_index.yaml > duplicates_index.yaml
+      - id: upload-artifact
+        name: Upload artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: duplicates-index
+          path: duplicates_index.yaml
+          retention-days: 2
+          if-no-files-found: error
+  deduplicate:
+    runs-on: ubuntu-latest
+    needs: merge
+    name: Deduplicate
+    steps:
+      - id: download-artifact
+        name: Download artifacts
+        uses: actions/download-artifact@v3
+        with:
+          name: duplicates-index
+      - id: remove
+        name: Remove duplicates
+        # Removes duplicates per entry using 'digest' as value.
+        run: yq eval '.entries[] |= unique_by(.digest)' duplicates_index.yaml > index.yaml
```

Comment:
I think there is not value on the "duplicated_index.yaml" intermediate one. Why not running both actions at the same step and get the full result?

Reply:
+1

Reply:
Done!